src/object_storage/objects.rs

```diff
@@ -19,13 +19,16 @@ use std::rc::Rc;
 
 use fallible_iterator::{FallibleIterator, IntoFallibleIterator};
 
+use osauth::services::OBJECT_STORAGE;
+
 use super::super::common::{
     ContainerRef, IntoVerified, ObjectRef, Refresh, ResourceIterator, ResourceQuery,
 };
 use super::super::session::Session;
 use super::super::utils::Query;
 use super::super::{Error, Result};
 use super::{api, protocol};
+use reqwest::Url;
```

Comment:
Please group this `use` with other 3rdparty imports (i.e. below `osauth`).

Reply:
Great remark, it is done in commit 7becce68f9d587e731ebb8df795d552f1f244cea.